driver/optmain.ml

```diff
@@ -239,6 +239,9 @@ let main () =
   try
     readenv ppf Before_args;
     Clflags.add_arguments __LOC__ (Arch.command_line_options @ Options.list);
+    Clflags.add_arguments __LOC__
+      ["-depend", Arg.Unit Makedepend.main_from_option,
+       "ARGS Compute dependencies between source files"];
```

Comment:
As my well-known hidden agenda is to get rid of `ocamldep`, I don't want to refer to it here...